gems/actionpack/6.0/patch.rbs

```diff
@@ -82,20 +75,3 @@ module Rack
     end
   end
 end
-
-# Remove the fake types for Gem::Version
-# if the real types are available.
-module Gem
-  class Version
-  end
-end
-
-# Remove the fake types for SimpleDelegator
-# if the real types are available.
-class SimpleDelegator
-end
-
-module Rails
-  class Engine
-  end
-end
```

Comment:
https://github.com/ruby/gem_rbs_collection/blob/main/gems/railties/6.0/railties-generated.rbs#L1947

Note: It seems not used. So I only removed this.